dev/Dockerfile.base

```diff
@@ -55,7 +56,8 @@ ENV PATH="/venv/bin:${PATH}" \
 COPY ./requirements/requirements.common.txt /tmp/requirements.txt
 
 RUN set -ex; \
-    pip install --no-cache-dir --upgrade pip \
+    chmod 2775 /venv/bin \
```

Comment:
Looking now, I'm wondering if this `chmod 2775 /venv/bin` is redundant, since /venv/ should already be 2775 and owned by USER_NAME.USER_GROUP.

I'll test and see if it's required.

Reply:
Yeah, the 'chmod 2775 /venv/bin' was unneeded.  Updated now.